redfishtoollib/redfishtoolMain.py

```diff
@@ -350,6 +354,10 @@ def main(argv):
         rft.subcommand=args[0]
         rft.subcommandArgv=list(args)
 
+    # disable reading of proxy environment variables for --no-proxy
+    if rft.no_proxy:
+        os.environ['NO_PROXY'] = '*'
```

Comment:
I think we already talked about this, but just to make sure, this environment setting only lives for the duration of the tool's execution?

Reply:
> I think we already talked about this, but just to make sure, this environment setting only lives for the duration of the tool's execution and is only visible in the context of the instance of the tool currently running?

That's correct - it only applies to the currently running Redfishtool process.